### PR TITLE
Reinvent Landing Page

### PR DESCRIPTION
**Pull request by nzoschke**
/reinvent landing page for the URL on the print flyers.

fireshot capture 7 - convox re_invent - http___localhost_reinvent

## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by nzoschke** [comment]
[Handle /reinvent Flyer URL](https://trello.com/c/GdOoAegK/1245-handle-reinvent-flyer-url)

File: reinvent/index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+						<a href="/docs/rack" target="_blank">Learn more ></a>
+					</p>
+				</div>
+			</div>
+			<div class="col-xs-12 col-md-6">
+				<div class="box">
+					<h4>Future-proof Architecture</h4>
+					<p>
+						Convox sets up a cloud service-oriented architecture.<br />
+						Every component is swappable, make it easy to update the architecture as AWS evolves.
```
Comment (soulshake) [inline reinvent/index.html]: s/make/making/

File: reinvent/index.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+					<p>
+						<a href="/blog/services-over-software" target="_blank">Read more ></a>
+					</p>
+				</div>
+			</div>
+			<div class="col-xs-12 col-md-6">
+				<div class="box">
+					<h4>Security Updates</h4>
+					<p>
+						The latest AMIs and Kernel patches are a `convox update` away.<br />
```
Comment (soulshake) [inline reinvent/index.html]: Can we use <pre> here instead of inline backticks?

File: reinvent/index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+						<a href="/docs/updating-rack-and-cli" target="_blank">Read the docs ></a>
+					</p>
+				</div>
+			</div>
+			<div class="col-xs-12 col-md-6">
+				<div class="box">
+					<h4>Community</h4>
+					<p>
+						Convox is <a href="https://github.com/convox/rack">open-source</a>.<br />
+						All the conversations about how use AWS best is out in the open.
```
Comment (soulshake) [inline reinvent/index.html]: s/how use/how to use/

File: reinvent/index.html (hunk 1 above)
Comment (nzoschke) [inline reinvent/index.html]: Fixed

File: reinvent/index.html (hunk 2 above)
Comment (nzoschke) [inline reinvent/index.html]: Good call. Fixed.

File: reinvent/index.html (hunk 3 above)
Comment (nzoschke) [inline reinvent/index.html]: Fixed

**Pull request merged**